src/main/scripts/process_dashboard.py

```diff
@@ -0,0 +1,99 @@
+import json
```

Comment:
Can we have some general comment/documentation what is the purpose of the script and what dashboards it can process (e.g. as Github links).

Also can we have a test for it? Just defining one URL and defining an expected output?

Reply:
Yes, we can test it for sure with a small dashboard json.

Reply:
Added a test which calls the script and then asserts a few things in the resulted json